packages/matter.js/src/protocol/interaction/InteractionClient.ts

```diff
@@ -215,26 +233,43 @@ export class InteractionClient {
         this.exchangeProvider.addProtocolHandler(new SubscriptionClient(this.subscriptionListeners));
     }
 
-    async getAllAttributes(): Promise<DecodedAttributeReportValue[]> {
-        return this.getMultipleAttributes([{ /* empty means *,*,* */ }]);
+    async getAllAttributes(): Promise<DecodedAttributeReportValue<any>[]> {
+        return (await this.getMultipleAttributesAndEvents([{ /* empty means "* / * /*"  */ }])).attributeReports;
```

Comment:
Maybe a constant instead of the comment, e.g. `const REQUEST_ALL = [{}]`?  With an object parameter then this could be `
```typescript
this.getMultipleAttributesAndEvents({ attributeRequests: REQUEST_ALL })`
```

Reply:
cool idea for the constant